setup.cfg

```diff
@@ -4,7 +4,7 @@ ignore =
 max-line-length = 100
 max-complexity = 18
 select = B,C,E,F,W,T4,B9
-extend-ignore = E203,E501,E402,W605
+extend-ignore = E203,E501,E402,W605,W503
```

Comment:
I actually don't need that one anymore, but note that that warning is incompatible with `black`